website/docs/lsp.md

```diff
@@ -154,6 +154,13 @@ A short list of useful LSP-related command line flags:
   diagnostics it sends to clients at 1,000 diagnostics, but this can be changed
   (set it to `0` to remove the cap).
 
+- `--cache=dir=...`
```

Comment:
Let's make sure to link to the cache-dir docs somewhere:

https://sorbet.org/docs/cli#--cache-dir-caching-parse-results